Applications/Welcome/TextWidget.cpp

```diff
@@ -0,0 +1,115 @@
+#include <AK/String.h>
+#include <AK/StringBuilder.h>
+#include <AK/Vector.h>
+#include <LibGUI/GPainter.h>
+
+#include "TextWidget.h"
+
+TextWidget::TextWidget(GWidget* parent)
+    : GFrame(parent)
+{
+}
+
+TextWidget::TextWidget(const StringView& text, GWidget* parent)
+    : GFrame(parent)
+    , m_text(text)
+{
+}
+
+TextWidget::~TextWidget()
+{
+}
+
+void TextWidget::set_text(const StringView& text)
+{
+    if (text == m_text)
+        return;
+    m_text = move(text);
```

Comment:
`move()` on a `const T&` does nothing -- you can move a `StringView` or a `StringView&&` but not a `const StringView&`.
```suggestion
    m_text = text;
```

Reply:
Hrmm, okay - I took that from GLabel, so it should probably be fixed there too.